src/main/kotlin/calculator/EquationTokenizer.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+class EquationTokenizer {
+
+    fun tokenize(equation: String): Pair<MutableList<Operand>, MutableList<Operator>> {
+        val splitEquation = equation.split(" ")
+        val operands = mutableListOf<Operand>()
+        val operators = mutableListOf<Operator>()
+
+        for ((index, element) in splitEquation.withIndex()) {
+            if (isOperandIndex(index)) {
+                operands.add(Operand(element))
+                continue
+            }
+            operators.add(Operator.of(element))
+        }
+
+        return Pair(operands, operators)
+    }
+
+    private fun isOperandIndex(index: Int) = index % 2 == 0
```

Comment:
코틀린은 확장함수라는것을 지원해서 이렇게 Int에 사용자가 함수를 추가해서 사용할수가 있습니다!

```suggestion
        for ((index, element) in splitEquation.withIndex()) {
            if (index.isOperand()) {
                operands.add(Operand(element))
                continue
            }
            operators.add(Operator.of(element))
        }

        return Pair(operands, operators)
    }

    private fun Int.isOperand() = this % 2 == 0
```

Reply:
오 ~ 확장함수를 잊고있었네용..
이번 리뷰와는 다른 이야기긴 한데 궁금한점이 있습니다..!  
비밥은 확장함수 사용할때, 그때 그때 필요한 클래스에 정의를 해서 쓰시는 편인가용? 
실제로 개발할때 확장함수를 어느 클래스파일에 언제 정의해서 쓸지 좀 고민되더라고요.. 로직이 뭔가 여기저기 흩뿌려지는 느낌이여서요. private 메서드 정도로 간주하고 사용하면 될까용?